python_src/src/lamp_py/migrations/versions/performance_manager_prod/001_5d9a7ee21ae5_initial_prod_schema.py

```diff
@@ -242,6 +242,25 @@ def upgrade() -> None:
         unique=False,
     )
 
+    op.create_table(
```

Comment:
should probably create a similar migration in the dev dir, right?

Reply:
I don't think we can. I'm not exactly sure how we'd deploy to dev at this point, since the schema changes combined with application logic to reflect those schema changes, has a lot of breaking changes with the existing dev schema.